server/games/game.py

```diff
@@ -861,17 +861,22 @@ def report_army_stats(self, stats_json):
         self._process_pending_army_stats()
 
     def is_visible_to_player(self, player: Player) -> bool:
+        """
+        Determine if a player should see this game in their games list.
+
+        Note: This is a *hot* function, it can have significant impacts on
+        performance.
```

Comment:
Is it because of the rating initialization? It doesn't become clear, why this comment on performance is here.

Reply:
It’s there because this is now the second time I’ve had to optimize something that was added to that function because it showed up in the profiler.